brainscore_language/__main__.py

```diff
@@ -1,350 +1,7 @@
-import argcomplete, argparse
-import brainscore_language as lbs
-import xarray as xr
-import wandb
-import os
+import fire
 
+# noinspection PyUnresolvedReferences
+from . import score
```

Comment:
a little uncomfortable here about the relative import :)

Reply:
i've just had so many bad experiences not doing a full module-level import
(`from brainscore_language.xyz import score`)

Reply:
happy to do absolute here